src/main/java/com/getify/minify/Minify.java

```diff
@@ -0,0 +1,103 @@
+//
+//  Minify.java
+//  jsonminifier - Ported from https://github.com/getify/JSON.minify
+//
+//  Created by Bernhard Gass on 8/01/13.
+//  Copyright © 2013 Bernhard Gass. All rights reserved.
+
+package com.getify.minify;
```

Comment:
Ah, I see. In that case, do you see yourself releasing more java code under `io.github.getify` and if so, is it OK for those projects to be associated under the same maven group as this project?